### PR TITLE
feat: action beard feedback on limit orders ui

### PR DESCRIPTION
**Pull request by woodenfurniture**
## Description

Actions feedback on the limit orders input UI.

- Breaks out the menu buttons for asset and chain selection for re-use in limit orders
- Fixes minor deviations from the product spec
- Adds interactions to set the limit, and to swap the price
- Updates the expiry dropdown items to match the design

All data is placeholder and is not wired up to anything meaningful.

## Issue (if applicable)

NA

## Risk

> High Risk PRs Require 2 approvals

Low risk

> What protocols, transaction types, wallets or contract interactions might be affected by this PR?

## Testing

Check the chain and asset menus in the trade input are still styled correctly.

### Operations

Check the chain and asset menus in the trade input are still styled correctly.

- [x] :checkered_flag: My feature is behind a flag and doesn't require operations testing (yet)

## Screenshots (if applicable)

https://jam.dev/c/c4c2efd2-9cfa-41cf-bd3c-b21d2c901a5c

### CODE REVIEW

File: src/components/AssetSelection/AssetSelection.tsx [hunk 1]
```diff
@@ -8,10 +8,8 @@ import { Text } from 'components/Text'
 import { selectAssets } from 'state/slices/selectors'
 import { useAppSelector } from 'state/store'
 
-import { AssetChainDropdown } from './components/AssetChainDropdown/AssetChainDropdown'
-import { AssetMenuButton } from './components/AssetMenuButton'
-
-const disabledStyle = { cursor: 'not-allowed' }
+import { StyledAssetChainDropdown } from './components/AssetChainDropdown/AssetChainDropdown'
```
Comment (gomesalexandre) [inline src/components/AssetSelection/AssetSelection.tsx:11]: q: what does `Styled` refer to across this PR?

File: src/components/AssetSelection/helpers.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import type { ButtonProps } from '@chakra-ui/react'
+
+const disabledStyle = { cursor: 'not-allowed' }
+
+export const getStyledMenuButtonProps = ({
```
Comment (gomesalexandre) [inline src/components/AssetSelection/helpers.ts:5]: same q. here

**Review by gomesalexandre — APPROVED** [review]
Tested locally, couldn't find anything weird here !
https://jam.dev/c/c23b4374-3932-4a71-9d83-50a5823d7671

File: src/components/AssetSelection/helpers.ts (hunk 2 above)
Comment (woodenfurniture) [reply src/components/AssetSelection/helpers.ts:5]: This helper is to get identical styling props to be shared across both component types without duplication

File: src/components/AssetSelection/AssetSelection.tsx (hunk 1 above)
Comment (woodenfurniture) [reply src/components/AssetSelection/AssetSelection.tsx:11]: I'll follow up with you directly on this one as likely you've got some magic up your sleeve to make this tidy

**Pull request merged**